go/vt/vttablet/tabletserver/throttle/client.go

```diff
@@ -109,12 +100,12 @@ func (c *Client) ThrottleCheckOK(ctx context.Context, overrideAppName throttlera
 	if checkResult.StatusCode != http.StatusOK {
 		return false
 	}
-	c.lastSuccessfulThrottle = atomic.LoadInt64(&throttleTicks)
```

Comment:
This was actually intentional. It is possible that by the time we reach this line, the ticker will have advanced even more. We prefer to use the very latest tick number. Not critical, but saves an iteration next time this is called.

Reply:
Ah ok, can change this!